# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1426109192 -3600
#      Wed Mar 11 22:26:32 2015 +0100
# Node ID 3c414ad460d5278d3f5ace5be9227c32055e879d
# Parent  928ac282b2fa2420f102963518ff104ba2f433b7
fix shortcut for previous bookmark (bug #44505)

* file-editor.cc (set_shortcuts): fixed setting wring shortcut to the
  previous bookmark action

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -1850,17 +1850,17 @@ file_editor::set_shortcuts ()
   shortcut_manager::set_shortcut (_show_longline_action, "editor_view:show_long_line");
   shortcut_manager::set_shortcut (_zoom_in_action, "editor_view:zoom_in");
   shortcut_manager::set_shortcut (_zoom_out_action, "editor_view:zoom_out");
   shortcut_manager::set_shortcut (_zoom_normal_action, "editor_view:zoom_normal");
 
   // Debug menu
   shortcut_manager::set_shortcut (_toggle_breakpoint_action, "editor_debug:toggle_breakpoint");
   shortcut_manager::set_shortcut (_next_breakpoint_action, "editor_debug:next_breakpoint");
-  shortcut_manager::set_shortcut (_previous_bookmark_action, "editor_debug:previous_breakpoint");
+  shortcut_manager::set_shortcut (_previous_breakpoint_action, "editor_debug:previous_breakpoint");
   shortcut_manager::set_shortcut (_remove_all_breakpoints_action, "editor_debug:remove_breakpoints");
 
   // Run menu
   shortcut_manager::set_shortcut (_run_action, "editor_run:run_file");
   shortcut_manager::set_shortcut (_run_selection_action, "editor_run:run_selection");
 
   // Help menu
   shortcut_manager::set_shortcut (_context_help_action, "editor_help:help_keyword");
